src/map.cpp

```diff
@@ -749,6 +755,53 @@ float map::vehicle_vehicle_collision( vehicle &veh, vehicle &veh2,
     return dmg_veh1;
 }
 
+bool map::check_vehicle_zones( const int zlev )
+{
+    for( auto veh : this->get_cache( zlev ).zone_vehicles ) {
```

Comment:
The `this->` is not needed (also appears at other places).

Reply:
map.cpp is full of `this->` but I'll fix the ones I added